src/ol/worker/geotiff-decoder.js

```diff
@@ -1,16 +1,14 @@
 import 'regenerator-runtime/runtime.js';
 
-// BEGIN copied from geotiff/src/decoder.worker.js
 import {Transfer, expose} from 'threads/worker';
-import {getDecoder} from 'geotiff/src/compression';
 
 async function decode(fileDirectory, buffer) {
+  const {getDecoder} = await import('geotiff/src/compression');
```

Comment:
This change does not directly relate to the goal of this pull request, but it reduces the size of the worker a bit. (90 kB vs. previously 92 kB).